andela/ah-backend-iroquois
# #159951992 Upgrade Authors Haven Django app from Django 1.10 to Django 2.1

#### What does this PR do?
- It upgrades the codebase from django ==1.10== to django 2.1

#### Description of Task to be completed?
- review all files to ensure that they are updated to match the new version

#### How should this be manually tested?
- Install django version 2.1
- After cloning the repo, cd into the directory and RUN ```python python manage.py runserver ```
- It should have resolved all errors and running the server

#### What are the relevant pivotal tracker stories?
[#159951992](https://www.pivotaltracker.com/story/show/159951992)
